bears/hypertext/HTMLHintBear.py

```diff
@@ -0,0 +1,180 @@
+import json
+
+from coalib.bearlib.abstractions.Linter import linter
+
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='htmlhint',
+        output_format='regex',
+        output_regex=r'(?P<filename>.+):(?P<line>\d+):(?P<column>\d+):\s*'
+                     r'(?P<message>.+). \[(?P<severity>error|warning)')
```

Comment:
you need to escape it then with a `\`. Also match the whole string not just until `error/warning`